training-node/test/users.js

```diff
@@ -0,0 +1,95 @@
+const chai = require('chai'),
+  dictum = require('dictum.js'),
+  server = require('./../app'),
+  should = chai.should();
+
+const successUserCreate = () =>
+  chai
+    .request(server)
+    .post('/users')
+    .send({ name: 'martin', lastName: 'picollo', email: 'martin@wolox.com.ar', password: '12346578q' });
+
+describe('users', () => {
+  describe('/users POST', () => {
+    it('should success creation ', done => {
+      successUserCreate().then(res => {
+        dictum.chai(res);
+        done();
+      });
+    });
+    it('should fail creation because of missing parameter password', done => {
+      chai
+        .request(server)
+        .post('/users')
+        .send({ name: 'martin', lastName: 'picollo', email: 'martin2@wolox.com.ar' })
+        .catch(err => {
+          err.should.have.status(400);
+          err.response.should.be.json;
+          err.response.body.should.have.property('error');
```

Comment:
Usually I also validate that the error message is correct, its no necessary for this training though

Reply:
I'll keep it in mind :+1: